packages/docusaurus-module-type-aliases/src/index.d.ts

```diff
@@ -319,6 +319,17 @@ declare module '@docusaurus/useGlobalData' {
   export default useGlobalData;
 }
 
+declare module '*.svg' {
+  import type {FunctionComponent, SVGProps} from 'react';
+
+  export const ReactComponent: FunctionComponent<
+    SVGProps<SVGSVGElement> & {title?: string}
```

Comment:
I would prefer not adding the `title` prop because afaik it's not std: usually you add it on the wrapping image instead. Also, `FunctionComponent` => `ComponentType`?

Reply:
Corrected in 6c677b9.